games/s.yaml

```diff
@@ -461,8 +461,9 @@
   originals:
   - 'Artemis: Spaceship Bridge Simulator'
   repo: https://github.com/smcameron/space-nerds-in-space
-  type: clone
-  updated: 2015-05-02
+  type: remake
```

Comment:
the type should be something else; remakes are for things like source ports, or engine reimplementations where the original assets can be used.